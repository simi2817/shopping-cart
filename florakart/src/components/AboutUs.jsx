import { Link } from "react-router-dom";
import Header from "./Header";
import Navigation from "./Navigation";

const AboutUs = () => {
    return (
        <>
            <Header />
            <Navigation />
            <div className="home">
                <img src='https://www.everwallpaper.co.uk/cdn/shop/products/cactus-and-flowers-wallpaper-mural-plain_cb1b71fe-8692-4ec8-87b2-23d0e717545a.jpg?v=1640140683' alt="flora" height={500}/>
                <br />
                <p>
                Looking to start or grow your collection of indoor plants?</p>
                <p>Whether you’re an experienced green-fingered guru or a first time plant parent, our range of potted house plants has something for everyone.</p>
                <p>From small succulents to powerful palms, flourishing ferns and charming calatheas.</p> 
                <p>The gang’s all here! Ready and waiting to turn your home into a gorgeous green indoor jungle.
                </p>
                <br />
                <Link to='/'>Back to home</Link>
            </div>
        </>
    );
}

export default AboutUs;