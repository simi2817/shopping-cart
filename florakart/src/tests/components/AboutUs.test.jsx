import { render } from '@testing-library/react';
import '@testing-library/jest-dom';
import { test, expect } from 'vitest';
import AboutUs from '../../components/AboutUs.jsx';
import { MemoryRouter } from 'react-router-dom';

test('renders a message', () => {
    const { container } = render(<MemoryRouter><AboutUs /></MemoryRouter>);
    expect(container).toMatchSnapshot();
})