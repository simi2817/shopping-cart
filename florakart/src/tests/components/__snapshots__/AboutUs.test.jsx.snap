// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`renders a message 1`] = `
<div>
  <h1
    class="header"
  >
    Flora-Kart
  </h1>
  <div
    class="navigation"
  >
    <nav>
      <a
        class="navLink"
        href="/about"
      >
        About-Us
      </a>
      <a
        class="navLink"
        href="/categories"
      >
        Categories
      </a>
      <a
        class="navLink"
        href="/cart"
      >
        Cart
      </a>
    </nav>
  </div>
  <div
    class="home"
  >
    <img
      alt="flora"
      height="500"
      src="https://www.everwallpaper.co.uk/cdn/shop/products/cactus-and-flowers-wallpaper-mural-plain_cb1b71fe-8692-4ec8-87b2-23d0e717545a.jpg?v=1640140683"
    />
    <br />
    <p>
      Looking to start or grow your collection of indoor plants?
    </p>
    <p>
      Whether you’re an experienced green-fingered guru or a first time plant parent, our range of potted house plants has something for everyone.
    </p>
    <p>
      From small succulents to powerful palms, flourishing ferns and charming calatheas.
    </p>
    <p>
      The gang’s all here! Ready and waiting to turn your home into a gorgeous green indoor jungle.
    </p>
    <br />
    <a
      href="/"
    >
      Back to home
    </a>
  </div>
</div>
`;
